security-analysis-api/src/main/java/com/powsybl/security/tools/SecurityAnalysisTool.java

```diff
@@ -209,7 +209,7 @@ static SecurityAnalysisInput buildPreprocessedInput(SecurityAnalysisExecutionInp
 
     private static SecurityAnalysisExecutionBuilder createBuilder(PlatformConfig platformConfig) {
         return new SecurityAnalysisExecutionBuilder(() -> ExternalSecurityAnalysisConfig.load(platformConfig),
-            () -> SecurityAnalysisFactories.newDefaultFactory(platformConfig),
+            platformConfig.getModuleConfig(MODULE_CONFIG_NAME_PROPERTY).getStringProperty(DEFAULT_SERVICE_IMPL_NAME_PROPERTY),
```

Comment:
is it possible to use the standard mechanism PlatformConfigNamedProvider.Finder here instead ? it does a little more than just looking at the field, when there is only one implementation it uses it without any configuration.

Reply:
Related to @sylvlecl comment